gmm_client/src/http_executor.erl

```diff
@@ -47,7 +47,7 @@ request(Method, Url, RawBody, GetResponse) ->
             _ -> client_utils:encode(RawBody)
         end,
     ReqHeaders = [{<<"Content-Type">>, <<"application/json">>}],
-    case hackney:request(Method, Url, ReqHeaders, Body) of
+    case hackney:request(Method, Url, ReqHeaders, Body, [{pool, false}]) of
```

Comment:
What does it do?

Reply:
it disables thread pools - without this, errors "checkout_timeout" happened when there were more than 40 vertices